src/migrations/migration_steps.ts

```diff
@@ -548,6 +548,20 @@ migrationStepRegistry
       }
       return data;
     },
+  })
+  .add("18.5.2", {
```

Comment:
Will probably be merged after https://github.com/odoo/o-spreadsheet/pull/6914, so the version should probably be increased.

Reply:
And an upgrade script will be required as well